content/react.md

```diff
@@ -0,0 +1,271 @@
+
+  - [`kadira:flow-router`](https://atmospherejs.com/kadira/flow-router) is a Meteor specific Router that can be used to render to React and that we document in the [Routing article](routing.html).
+
+  - [`react-router`](https://www.npmjs.com/package/react-router) is a pure-React Router that is very popular and which can also be used easily with Meteor.
+
+<h3 id="using-flow-router">Flow Router</h3>
+
+Using Flow Router with React is almost exactly analogous to using it with Blaze. The only difference is that in your route actions, you should use the [`react-mounter`](https://www.npmjs.com/package/react-mounter) package to mount components into your layout. Once you `npm install --save react-mounter`, you can:
+
+```jsx
+import { FlowRouter } from 'meteor/kadira:flow-router';
+import { mount } from 'react-mounter';
+
+import AppContainer from '../../ui/containers/AppContainer.jsx';
+import ListContainer from '../../ui/containers/ListContainer.jsx';
+
+
+FlowRouter.route('/lists/:_id', {
+  name: 'Lists.show',
+  action() {
+    mount(AppCountainer, {
+      main: () => <ListContainer/>,
+    });
+  },
+});
+```
+
+Note that `react-mounter` automatically mounts the layout component on a `#react-root` node, which you can change by using the `withOptions()` function.
+
+<h3 id="using-react-router">React Router</h3>
+
+Using React Router is also straightforward. Once you `npm install --save react-router`, you can simply export a list of nested routes as you would in any other React Router driven React application:
+
+```jsx
+import React from 'react';
+import { Router, Route, browserHistory } from 'react-router';
+
+// route components
+import AppContainer from '../../ui/containers/AppContainer.jsx';
+import ListContainer from '../../ui/containers/ListContainer.jsx';
+import AuthPageSignIn from '../../ui/pages/AuthPageSignIn.jsx';
+import AuthPageJoin from '../../ui/pages/AuthPageJoin.jsx';
+import NotFoundPage from '../../ui/pages/NotFoundPage.jsx';
+
+export const renderRoutes = () => (
+  <Router history={browserHistory}>
+    <Route path="/" component={AppContainer}>
+      <Route path="lists/:id" component={ListContainer}/>
+      <Route path="signin" component={AuthPageSignIn}/>
+      <Route path="join" component={AuthPageJoin}/>
+      <Route path="*" component={NotFoundPage}/>
+    </Route>
+  </Router>
+);
+```
+
+With React Router, you'll need to render the exported routes in a startup function:
+
+```jsx
+import { Meteor } from 'meteor/meteor';
+import { render } from 'react-dom';
+import { renderRoutes } from '../imports/startup/client/routes.jsx';
+
+Meteor.startup(() => {
+  render(renderRoutes(), document.getElementById('app'));
+});
+```
+
+In general routing with React Router in Meteor follows generally the [same principles](routing.html) as using Flow Router, however you'll need to follow some of the idioms outlined in the [documentation](https://github.com/reactjs/react-router/blob/latest/docs/Introduction.md).
```

Comment:
Would be nice to have a summary of some of the different idioms for people who are trying to decide between Flow Router and React Router.